src/CMake/nativeWin.cmake

```diff
@@ -31,6 +31,11 @@ include_directories(${Boost_INCLUDE_DIRS})
 add_compile_definitions("BOOST_LOCALE_HIDE_AUTO_PTR")
 add_compile_definitions("BOOST_BIND_GLOBAL_PLACEHOLDERS")
 
+# Protobuf Libraries
+# Don't include finding protobuf module here as Windows build doesn't specify
+# where to the path to where the prebuilt protobuf is installed yet
+set(ENV{XRT_PROTOBUF_INSTALL} "${PROTOBUF_ROOT}")
```

Comment:
I think this should be passed in through windows build script a CMake configuration variable so that find_package can be used.